WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1830,6 +1827,24 @@ protected void onPostExecute(Uri uri) {
         }
     }
 
+    /**
+     * Media
+     */
+
+    private void addAllMedia(List<Uri> mediaUris) {
+        for (Uri mediaUri : mediaUris) {
+            if (mediaUri == null) {
```

Comment:
Since both conditions show the same toast, we could make this `if (mediaUri == null || !addMedia(mediaUri))`.